.github/workflows/test.yml

```diff
@@ -51,6 +52,28 @@ jobs:
       - name: Run tests
         run: pytest
 
+      - name: Test CLI
+        run: graphium --help
+
+      - name: Test building the doc
+        run: mkdocs build
+  
+  codecov:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.10"]
+        pytorch-version: ["1.13"]
```

Comment:
If graphium is compatible with pytorch 2 then I propose to test for pytorch 2 here (optionally we can test for both 1.13 and 2.

@DomInvivo any opinion?

Reply:
I've updated this to 2.0 here as well.